debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -611,6 +623,12 @@ else if (data instanceof java.util.Date) {
             // back to GMT
             data = OffsetTime.ofInstant(Instant.ofEpochMilli(((Date) data).getTime()), ZoneOffset.UTC);
         }
+        else if (data instanceof String) {
```

Comment:
So given the change below doesn't it render the handling of `java.util.Date` above obsolete now?